app/content_script.css

```diff
@@ -1,4 +1,4 @@
-#interceptor-container {
+.interceptor-container {
   position: absolute;
   top: 10px;
   right: 10px;
```

Comment:
I think this one is being used in the content script. Since I didn't remove the code, I'm keeping this as it is